tests/plugins/pastefromwordimage/helpers.js

```diff
@@ -0,0 +1,84 @@
+/* bender-tags: editor,pastefromwordimage */
+/* bender-ckeditor-plugins: pastefromwordimage */
+/* bender-include: _fixtures/helpers-fixtures.js */
+/* global fixtures */
+
+( function() {
+	'use strict';
+
+	bender.test( {
+		'test img tags extraction from html string': function() {
+			var item,
+				i,
+				length,
+				test,
+				result;
+
+			test = [
+				{
+					htmlString: '<p><img src="http://example-picture.com/random.png" /></p>',
+					result: [
+						[ '<img src="http://example-picture.com/random.png', 'http://example-picture.com/random.png' ]
+					]
+				},
+				{
+					htmlString: '<p><img class="border" width="400" height="300" data-custom="something" src="http://example-picture.com/random.png?parameter=2" /></p>',
+					result: [
+						[ '<img class="border" width="400" height="300" data-custom="something" src="http://example-picture.com/random.png?parameter=2',
+							'http://example-picture.com/random.png?parameter=2' ]
+					]
+				},
+				{
+					htmlString: '<p><img src="file://Some/Path/To/Local/Resources/kitty.jpeg" class="invisible" /></p>',
+					result: [
+						[ '<img src="file://Some/Path/To/Local/Resources/kitty.jpeg', 'file://Some/Path/To/Local/Resources/kitty.jpeg' ]
+					]
+				},
+				{
+					htmlString: '<p><img class="one" src="//example.com/pic1.png" /></p><p>Hello world</p><p><img src="//example.com/pic2.gif" class="two" /></p>' +
+						'<div><span class="something"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAAXNSR0IArs4c6QAAAAxJREFUCNdjYGBgAAAABAABJzQnCgAAAABJRU5ErkJggg==" /></span></div>',
+					result: [
+						[ '<img class="one" src="//example.com/pic1.png', '//example.com/pic1.png' ],
+						[ '<img src="//example.com/pic2.gif', '//example.com/pic2.gif' ],
+						[ '<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAAXNSR0IArs4c6QAAAAxJREFUCNdjYGBgAAAABAABJzQnCgAAAABJRU5ErkJggg==',
+							'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAAXNSR0IArs4c6QAAAAxJREFUCNdjYGBgAAAABAABJzQnCgAAAABJRU5ErkJggg==' ]
+					]
+				},
+				{
+					htmlString: '<p>There is no img</p>',
+					result: []
+				}
+			];
+
+			while ( item = test.shift() ) {
+				result = CKEDITOR.plugins.pastefromwordimage.extractImgTagsFromHtmlString( item.htmlString );
+				assert.areSame( item.result.length, result.length );
+				for ( i = 0, length = result.length; i < length; i++ ) {
```

Comment:
It's not really worth to extract this `length` variable. It decreases the readability.